cli/template/extras/config/postcss.config.cjs

```diff
@@ -1,6 +1,8 @@
-module.exports = {
+const config = {
```

Comment:
dont think there is a type we can use here but oh well - might as well be consistent

Reply:
Wonder if `/** @type {import('postcss-load-config').Config */` would work:

* https://github.com/matrix-org/matrix-authentication-service/blob/64ce271d08a8b38f03ff0317d2f04dd69c264f5f/frontend/postcss.config.cjs#L15
* https://github.com/vnphanquang/svelte-put/blob/e625fe8a794b8aee4aa811d960ea0448d686cb5d/sites/docs/postcss.config.cjs#L5